ui/src/services/graphUtils.js

```diff
@@ -180,3 +196,25 @@ export const getSeriesForSymmetricalChart = (
   }
   return series;
 };
+
+export const getNodesInterfacesString = (nodeIPsInfo): string => {
+  const planeInterfaces = [];
+
+  for (const planeIP of Object.values(nodeIPsInfo)) {
+    const controlPlaneInterface = planeIP?.controlPlane?.interface;
+    const workloadlPlaneInterface = planeIP?.workloadPlane?.interface;
+    if (
+      planeInterfaces.indexOf(controlPlaneInterface) === -1 &&
+      controlPlaneInterface
+    ) {
+      planeInterfaces.push(controlPlaneInterface);
+    }
+    if (
+      planeInterfaces.indexOf(workloadlPlaneInterface) === -1 &&
+      workloadlPlaneInterface
+    ) {
+      planeInterfaces.push(workloadlPlaneInterface);
+    }
+  }
+  return planeInterfaces.join('|');
```

Comment:
```suggestion
  const interfaces = Object.values(nodeIPsInfo).flatMap(planes => [plane?.controlPlane?.interface, plane?.networkPlane?.interface]);
  const uniqueInterfaces = [...new Set(interfaces)];
  return uniqueInterfaces.join('|'); //To me this join should be performed by the component and this function returns unique interfaces because we might want to get unique interfaces elesewhere whereas the join('|') is really coupled to the current usecase
```